client/state/post-formats/schema.js

```diff
@@ -0,0 +1,19 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		// Site ID
+		'^\\d+$': {
+			type: 'object',
+			description: 'List of supported post formats.',
+			additionalProperties: false,
+			patternProperties: {
+				// ID of the post format
+				'^[0-9a-z]+$': {
```

Comment:
Good catch there. Post formats are actually taxonomy terms, and their ID is actually the term slug. Term slugs are limited by [`sanitize_title_with_dashes()`](https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-includes/formatting.php#L1579), which means they can also contain dashes and underscores.